Add Getting Started File (#96)
WomenWhoCode/WWCodeGreenville: Gettingstarted.md

# How to contribute

To contribute to this project, please follow these steps:

1.**Star  this repository** (Click the **Star** button at the top of this page)

![Point to a star button](/assets/readme/StarRepo.png)

2.**Fork this repository** (You can do this by clicking the **Fork** button on this repository's page)

![Point to a fork button](/assets/readme/forkRepo.png)

> **Clone this repository** to your local manchine. Open a termianl navigate to the directory where you want to clone this repository. Then, run the following command:

Reviewer: ```suggestion
**Clone this repository** to your local machine. Open a terminal and navigate to the directory where you want to clone this repository. Then, run the following command:
```

> `https://github.com/<Your_username>/WWCodeGreenville.git`

Reviewer: ```suggestion
`https://github.com/<your_username>/WWCodeGreenville.git`
```

> Replace <your_username> with your GitHub username

Reviewer: ```suggestion
Replace `<your_username>` with your GitHub username
```

3.**Create a new branch**. To do this, run the following command:

`git checkout -b <branch_name>`

> Replace <branch_name> with the name of the branch that you want to create.

Reviewer: ```suggestion
Replace `<branch_name>` with the name of the branch that you want to create.
```

4.**Make your changes**. Open the file(s) that you want to edit in any code editor and make your changes.

5.**Commit your changes**. To do this, run the following command:

`git commit -m "<commit_message>"`

> Replace <commit_message> with a short message that describes your changes.

Reviewer: ```suggestion
Replace `<commit_message>` with a short message that describes your changes.
```

6.**Push your changes to your forked repository**. To do this, run the following command:

`git push origin <branch_name>`

> Replace <branch_name> with the name of the branch that you want to pushed your changes to.

Reviewer: ```suggestion
Replace `<branch_name>` with the name of the branch that you want to pushed your changes to.
```

> 7.**Create a pull request**. Go to the repository page that you forked and click the **Compare and pull request**.

Reviewer: ```suggestion
7.**Create a pull request**. Go to the repository page that you forked and click the **Compare and pull request** button
```

![Point to a Pull request button](/assets/readme/compare-and-pull-request.png)

> 8.**Respond to feedback**. we may review your changes and suggest improvements. so try to Respond to our feedback and make any necessary changes.

Reviewer: ```suggestion
8.**Respond to feedback**. We may review your changes and suggest improvements, so try to respond to our feedback and make any necessary changes.
```

> 9.**Merge your pull request**. Once we're happy with your changes, they we will merge your pull request into the main branch of the repository.

Reviewer: ```suggestion
9.**Merge your pull request**. Once we're happy with your changes, we will merge your pull request into the main branch of the repository.
```
